plugin/src/test/groovy/org/gradle/testretry/testframework/TestNGViaJUnitEngineFuncTest.groovy

```diff
@@ -55,9 +58,12 @@ class TestNGViaJUnitEngineFuncTest extends BaseTestNGFuncTest {
 
     @Override
     String reportedParameterizedMethodName(String gradleVersion, String methodName, String paramType, int invocationNumber, @Nullable String paramValueRepresentation) {
-        GradleVersion.version(gradleVersion) > GRADLE_5_4_1
-            ? "${methodName}(${paramType}) > [${invocationNumber}] ${paramValueRepresentation ?: ''}"
-            : "${methodName}(${paramType})[${invocationNumber}]"
+        switch (GradleVersion.version(gradleVersion)) {
+            case { it < GRADLE_6_1 }: return "${methodName}(${paramType})[${invocationNumber}]"
+            case { it < GRADLE_7_0 }: return "[${invocationNumber}] ${paramValueRepresentation ?: ''}"
+            case { it < GRADLE_8_1 && it != GRADLE_7_6_4 }: return "${methodName}(${paramType})[${invocationNumber}]"
```

Comment:
Wouldn't it be more resilient to use `it < GRADLE_8_1 && !(it >= GRADLE_7_6_4 && it < GRADLE_8_0)` as a condition?

Reply:
I doubt there will be another 7.6.x release, so let's try to keep it simple for now.